_posts/2018-12-24-ApiaryUI.md

```diff
@@ -0,0 +1,131 @@
+
+ApiaryUI is implemented in a way, that allows complete separation from [apiary.io](https://apiary.io/)
+and possibly forming a new separate product in the future. Some features from v3 are still missing
+in ApiaryUI. Some requested features not present in v3 were implemented to ApiaryUI.
+We’ve managed to fix tons of v3 bugs inside the new ApiaryUI. 
+
+As the architecture of ApiaryUI is very simple and solid, we are able to add new features
+and fix bugs in very rapid pace. Our extensive suite of unit, integration and e2e tests
+are helping with that. This can finally release further evolution of [API Blueprint](https://apiblueprint.org/) language.
+
+We are currently in a process of creating the deprecation strategies for  
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/)
+and [Embed](https://help.apiary.io/tools/embed/). When we have those strategies fully formed
+we'll them announce on [Apiary Blog](https://blog.apiary.io/).
+
+
+### What's new in ApiaryUI
+
+ApiaryUI went through major product decisions. The concept of having three column layout stayed,
+but we reorganized the information displayed on each of them. Human Column is now designed
+to contain all the information needed for API Viewer to fully understand what the API is all about.
+Machine Columns sole responsibility is to display machine processing related information
+about one Http Transaction - Response/Request pair.
+
+Although ApiaryUI exclusively uses [API Elements](https://apielements.org/) as data format, it can also consume
+Api Description Document (ADD) formats like [API Blueprint](https://apiblueprint.org/) or [OpenAPI 2.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) ([OpenAPI 3.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md) coming soon)
+and it parses the ADDs in Web Worker inside the browser. This means that ApiaryUI is completely
+backend-less and does not need the server as it's backend.
+
+#### Stability and predictability
+
+ApiaryUI is React/Redux SPA (Single Page Application). We've used industry standard architectures
+and patterns to produce simple and standardized codebase. As I mentioned earlier, our extensive
+tests suite guarantees the stability of the ApiaryUI and makes further development predictable. 
+
+
+#### Code highlighting
+
+ApiaryUI highlights code examples in the following languages:
+
+- Javascript
+- JSON
+- XML
+- Python
+- Ruby
+- Java
+
+#### Print support
+
+ApiaryUI is fully print friendly. You can easily print the rendered documentation directly on paper or into PDF.
+
+
+#### Accessibility
+
+ApiaryUI complies with latest accessibility standards. We care about correctness of HTML semantics and proper use of Aria attributes.
+ApiaryUI is compatible with screen readers and even specially-abled people should navigate in it with no problems.
```

Comment:
TIL specially-abled, although [some people do not like it](https://adventuresbeyondbarriers.com/tag/specially-abled/).

Reply:
Indeed. Even our CEO refers to these users as _disabled_  in http://www.oracle.com/us/corporate/accessibility/index.html.

BTW accessibility isn't only meant for disabled users, it also serves power users.

I would rephrase to something like:

> ApiaryUI is supports the usage of screen readers and keyboard navigation. Whether you have a disability, or a personal usage preference, it allows you to choose how to interact with the interface.

Reply:
And probably I would drop `ApiaryUI complies with latest accessibility standards`. We don't comply yet with minimum contrast ratio and some other nuances of the latest standards.

Reply:
Used disabled, dropped some sentences and rephrase the thing.